lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -1385,6 +1385,12 @@ void SessionState::get_charging_updates(
         }
         grant->set_service_state(SERVICE_REDIRECTED, *credit_uc);
         action->set_redirect_server(grant->final_action_info.redirect_server);
+        // activate service
```

Comment:
this comment seems unrelated. nit ?

Reply:
`ACTIVATE_SERVICE` below is defined as `action->set_ambr(config_.get_apn_ambr());` 

Since with the refactor we lose the label I want to make sure that we know what this command is doing. Per its name it is not evident it is causing an activation so we better leave a comment for our future selves 

Same with the `//termination service`